stages/org.osbuild.container-image.install

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python3
+"""
+Install an container image into the container store.
+This supports both oci archives and docker archives, and can use both the container-image
+source or a file source (for example with the output of an org.osbuild.oci-archive stage.
+
+Buildhost commands used: `skopeo`.
+"""
+
+import os
+import subprocess
+import sys
+import tempfile
+
+import osbuild.api
+
+
+SCHEMA_2 = r"""
+"inputs": {
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["images"],
+  "properties": {
+    "images": {
+      "type": "object",
+      "additionalProperties": true
+    }
+  }
+},
+"options": {
+  "additionalProperties": false,
+  "properties": {
+    "storage-path": {
+      "description": "Container storage location (default /var/lib/containers/storage).",
+      "type": "string"
+    },
+    "storage-driver": {
+      "description": "The container storage driver to use (default overlay).",
+      "type": "string"
+    }
+  }
+}
```

Comment:
If we wrap this in a "destination" we could in the future also support copying the containers to random locations in the tree in oci formats. Not sure this is need (maybe onto an iso?), but it would be neat to have to option.

```
"destination": {
   "type": "containers-storage`, // type enum: ["containers-storage", "oci-archive"]
   ... options ..
}